src/content/app/genome-browser/components/browser-sidebar-modal/services/browser-sidebar-modal-storage-service.ts

```diff
@@ -18,14 +18,14 @@ import storageService, {
   StorageServiceInterface
 } from 'src/services/storage-service';
 
-import { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/track-panel/trackPanelSlice';
+import { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/browser-sidebar-modal/browserSidebarModalSlice';
 
 export enum StorageKeys {
-  BOOKMARKS = 'trackPanel.bookmarks',
-  PREVIOUSLY_VIEWED = 'trackPanel.previouslyViewedObjects'
+  BOOKMARKS = 'browserSidebarModal.bookmarks',
+  PREVIOUSLY_VIEWED = 'browserSidebarModal.previouslyViewedObjects'
```

Comment:
It's really just a UI accident that bookmarks or previously viewed objects are being shown in a modal; the key in the storage shouldn't care. Should think about how better to name this. Perhaps even `genomeBrowser.bookmarks`.

Reply:
✅